.env

```diff
@@ -13,6 +13,9 @@ STACKS_BLOCKCHAIN_API_HOST=127.0.0.1
 STACKS_CORE_RPC_HOST=127.0.0.1
 STACKS_CORE_RPC_PORT=20443
 
+# STACKS_CORE_PROXY_HOST=127.0.0.1
```

Comment:
Just double checking but this ENV can accept a FQDN, right?

Reply:
Yep 👍